process_sentinel.py

```diff
@@ -362,24 +362,24 @@ def set_default_options(inps, template):
                                      inps_name = template_val.inps_name,
                                      default_value= template_val.default_value)
 
-def call_ssara(custom_template_file, slcDir):
 
-        out_file = os.getcwd() + '/' + 'out_download_ssara' 
-        command = 'download_ssara_rsmas.py ' + custom_template_file 
+def call_ssara(custom_template_file, slcDir):
+        out_file = os.getcwd() + '/' + 'out_{script_name}'
+        command = '{script_name}.py ' + custom_template_file
         #command = '('+command+' | tee '+out_file+'.o) 3>&1 1>&2 2>&3 | tee '+out_file+'.e'  # not used because it only works in bash
-        messageRsmas.log(command)
-        command = '('+command+' > '+out_file+'.o) >& '+out_file+'.e' 
-        command_ssh = 'ssh pegasus.ccs.miami.edu \"s.cgood;cd ' + slcDir + '; ' +  command + '\"'
-        status = subprocess.Popen(command_ssh, shell=True).wait()
-        print('Exit status from download_ssara_rsmas.py:',status)
-
-        out_file = os.getcwd() + '/' + 'out_download_asfserial' 
-        command = 'download_asfserial_rsmas.py ' + custom_template_file 
-        messageRsmas.log(command)
-        command = '('+command+' > '+out_file+'.o) >& '+out_file+'.e' 
-        command_ssh = 'ssh pegasus.ccs.miami.edu \"s.cgood;cd ' + slcDir + '; ' +  command + '\"'
-        status = subprocess.Popen(command_ssh, shell=True).wait()
-        print('Exit status from download_asfserial_rsmas.py:',status)
+        command_with_piping_output = '('+command+' > '+out_file+'.o) >& '+out_file+'.e'
+        command_ssh = 'ssh pegasus.ccs.miami.edu \"' \
+                      's.cgood;' \
+                      'cd ' + slcDir + '; ' + \
+                      command_with_piping_output + '\"'
```

Comment:
Why does `command_with_piping_output` need to be its own variable here? I think it would be more readable and clear if not separated out into a separate variable.

Reply:
Either way works. I think in general it makes the code clearer as to what change as made to that variable. In general, I am wary of this pattern:

```
variable = ....
variable = f(variable)
```
where `f` modifies `variable`

Reply:
I think I disagree. You can know what `command_ssh` does at a high level from just that variable definition. If you want to understand what that command is doing internally, you can look back at `command`.

However, I don't think that relates to renaming the variable though.

Reply:
I can foresee issues arising in the future where having to jump between variables gets confusing.

Reply:
I don't see how having the variable called the same thing would change anything? You're looking within the same 3 lines for either something with the same variable name or a different variable name.

I'm happy to debate based on general readability principles or standards, but I think the more prevalent standard here is mutation of a variable makes it unclear what the generic variable name (in this case, `command` haha).

Reply:
I think you can just get rid of the `command_with_piping_output` variable entirely. It's only used in one place, and I don't see any readability increase by separating it.

Reply:
Good point. I think combining that and @falkamelung's idea into making it into its own function is a good idea